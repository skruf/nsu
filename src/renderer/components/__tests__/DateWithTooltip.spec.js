import createLocalVue from "@/utils/tests/createLocalVue"
import { mount } from "@vue/test-utils"
import { Tooltip } from "element-ui"
import VueMoment from "vue-moment"
import DateWithTooltip from "@/components/DateWithTooltip"

const localVue = createLocalVue([
  VueMoment,
  Tooltip
])

describe("DateWithTooltip", () => {
  it("should render the component", () => {
    const propsData = { date: "" }
    const wrapper = mount(DateWithTooltip, { localVue, propsData })
    expect(wrapper.html()).toMatchSnapshot()
  })

  it("should display dates and format them properly", () => {
    const propsData = { date: "2019-01-23T20:42:03.713Z" }
    const wrapper = mount(DateWithTooltip, { localVue, propsData })
    const tooltip = wrapper.find(DateWithTooltip)
    expect(tooltip.exists()).toBe(true)
    // expect(tooltip.text()).toBe("24.01.19")
  })
})
