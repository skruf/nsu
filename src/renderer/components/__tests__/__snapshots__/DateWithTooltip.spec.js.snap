// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateWithTooltip should render the component 1`] = `undefined`;
